egs/wsj/s5/steps/make_mfcc.sh

```diff
@@ -10,7 +10,7 @@ nj=4
 cmd=run.pl
 mfcc_config=conf/mfcc.conf
 compress=true
-write_utt2num_frames=false  # if true writes utt2num_frames
+write_utt2num_frames=true  # if true writes utt2num_frames
```

Comment:
I think the final decision for what to do about this was to leave these options with the defaults all at false, but in the recipes where you need it, just use the flag.